equipment_handler/include/common_defines.h

```diff
@@ -37,21 +37,21 @@
 // Voltage/Current Sensor Constants
 #define ADC_VOLT_MAX            2.5f        //V
 #define ADC_VOLT_MIN            0           //V
-#define VOLT_MAX                5.44f       //V
+#define VOLT_MAX                16.32f       //V
 #define VOLT_MIN                0           //V
-#define CURR_MAX                1.437f      //A
+#define CURR_MAX                0.6f      //A
 #define CURR_MIN                0           //A
 
 // Slave Address for each panel of the hyperion board
 #define PANEL_SLAVE_ADDR_PORT                       0x20
 #define PANEL_SLAVE_ADDR_PORT_DEPLOYABLE            0x22
 #define PANEL_SLAVE_ADDR_STARBOARD                  0x23
-#define PANEL_SLAVE_ADDR_STARTBOARD_DEPLOYABLE      0x28
+#define PANEL_SLAVE_ADDR_STARBOARD_DEPLOYABLE       0x28
 #define PANEL_SLAVE_ADDR_ZENITH                     0x2E
-#define PANEL_SLAVE_ADDR_PORT2U                     0x20
+#define PANEL_SLAVE_ADDR_PORT2U                     0x2A
```

Comment:
This is wrong. I forgot to revert my testing code - 0x2A was just for one of our test boards. 0x20 is correct.